console/types/group/src/lib.rs

```diff
@@ -90,10 +90,6 @@ impl<E: Environment> Group<E> {
 
     /// Returns `self / COFACTOR`.
     pub fn div_by_cofactor(&self) -> Self {
-        // (For advanced users) The cofactor for this curve is `4`. Thus doubling is used to be performant.
```

Comment:
This was deleted because it was a copypasta from `mul_by_cofactor` (the method above).

The `mul_by_cofactor_inv` does not use this trick/assumption (it does the full `mul_bits`).